survey-runner-routing/vpc_endpoint.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_vpc_endpoint" "private-dynamodb" {
+  vpc_id       = "${var.vpc_id}"
+  service_name = "com.amazonaws.${data.aws_region.current.name}.dynamodb"
+  route_table_ids = ["${aws_route_table.private.*.id}"]
+}
+
+resource "aws_vpc_endpoint" "private-s3" {
```

Comment:
All of our other resources use `_` rather than `-`, so this should be `private_s3`